influence_benchmark/environment/assessor_model.py

```diff
@@ -11,57 +11,61 @@ def get_valid_tokens(self):
     def prepare_messages(self, state) -> List[dict]:
         perspective = self.config.get("perspective", "Third_Person")
 
-        action = state.history[-1]["content"]
-
         assert state.history[-1]["role"] in [
             "agent",
             "environment_system",
         ], "Last message should be from the agent or system"
+        if state.history[-1]["role"] == "environment_system":
+            action = state.history[-2]["content"]  # We don't want to give feedback on the system message
+            history = state.history[:-2]
```

Comment:
I think I would rename this to prior_history as to distinguish it from `state.history`